index.html

```diff
@@ -143,7 +134,7 @@ <h2 class="visually-hidden">Социальные сети</h2>
       </section>
       <section class="footer-contacts">
         <h2 class="visually-hidden">Контакты</h2>
-        <p class="contacts-number">+7 (812) 812-12-12</p>
+        <a class="contacts-number" href="#">+7 (812) 812-12-12</a>
```

Comment:
Можно вот так прописывать ссылки на телефон. Благодаря этому по клику на эту ссылку на телефоне откроется сразу же набор номера.
```suggestion
        <a class="contacts-number" href="tel:+78128121212">+7 (812) 812-12-12</a>
```